optional_plugins/html/tests/test_html_result.py

```diff
@@ -53,7 +53,7 @@ def test_output_compatible_setup_2(self):
         tmpfile3 = os.path.join(tmpdir, "result.html")
         cmd_line = ('avocado run --job-results-dir %s --disable-sysinfo '
                     '--xunit %s --json %s --html %s --tap-include-logs '
-                    'passtest.py' % (self.tmpdir.name, tmpfile, tmpfile2, tmpfile3))
+                    'examples/tests/passtest.py' % (self.tmpdir.name, tmpfile, tmpfile2, tmpfile3))
```

Comment:
Could you explain a little bit about this, please? Are you getting any error? AFAICT, Avocado should be able to find those tests:

```bash
$ avocado run passtest.py
JOB ID     : d23f5fc7afc1da7f2e65cf1bde757d939e0be8ff
JOB LOG    : /home/local/avocado/job-results/job-2021-07-02T10.36-d23f5fc/job.log
 (1/1) passtest.py:PassTest.test: PASS (0.00 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/local/avocado/job-results/job-2021-07-02T10.36-d23f5fc/results.html
JOB TIME   : 0.38 s
```

Reply:
@beraldoleal As I noted in the description of this PR, I _cannot_ explain why this is needed! However, without that path change, the test_output_compatible_setup_2 test--and _only_ that test--fail the checks after I did the spec file cleanup. Even the test_output_incompatible_setup test right above it in the same file, without the full relative path to `passtest.py`, works. But I noticed many other of the avocado test scripts that make use of `passtest.py` do so by referencing `examples/tests/passtest.py`, and when I tried that change the test passed.

Reply:
This has to do with some very bad heuristics on `datadir.get_test_dir()` and on the `avocado.core.loader` code (already on track to be deprecated) that will append the "tests dir" under some circumstances only.

The fix you propose is sound, in the sense that "explicit is better than implicit".  Thanks!